CMakeLists.txt

```diff
@@ -19,13 +19,7 @@
 ############################################################################
 
 PROJECT(mudlet)
-
-# Windows builds need 2.8.11, but for other platforms 2.8.9 should be enough
-IF(MSVC)
-  CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11)
-ELSE()
-  CMAKE_MINIMUM_REQUIRED(VERSION 2.8.9)
-ENDIF()
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
```

Comment:
Are you absolutely sure you HAVE to have 3.1 on all platforms (given that we only have 2.8.11.2 on the Linux Travis CI Platform)